docs/install/lotus/requirements.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Requirements
+description: There are specific system and hardware requirements for running a Lotus node or mining set up.
+---
+
+# Requirements
+
+The requirements of your system will change depending on the type of client you are running: a basic node, a storage miner, or seal worker.
+
+## Hardware
+
+### Lotus node
+
+Here's what you need to run a single lotus node for non-mining functions such as chain syncing, wallet transactions, storage or retrieval client, or building on Filecoin. (If you plan to mine, see the additional [Storage miner]{#storage-miner) requirements below.)
+
+The requirements table below is correct as of August 2020 but may change in the future as software development continues. The amount of memory (RAM) required is expected to drop significantly to about 1GB in the future as the Lotus client uses Badger FS more efficiently.
+
+| Processor                                      | Memory   | Storage space         |
+| ---------------------------------------------- | -------- | --------------------- |
+| 4 cores<br>8 threads<br> 2.5GHz base frequency | 8GB DDR4 | 200GB (SSD preferred) |
+
+### Storage miner
+
+Miners require a significant amount of hardware to run. The setup below is a minimal example for sealing 32GiB sectors on Lotus:
+
+| Processor                                      | Memory     | Storage space |
+| ---------------------------------------------- | ---------- | ------------- |
+| 4 cores<br>8 threads<br> 2.5GHz base frequency | 128GB DDR4 | 2TB           |
```

Comment:
128GB RAM is the minimal only if you add additional 256GB of very fast swap mounted on a NVMe SSD. Might mention that. Things break without it.